### PR TITLE
Update 3d-viewer version

### PR DESCRIPTION
**Pull request by mrtraser**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-219.gearboxjs.preview.cogniteapp.com

File: src/components/Model3DViewer/Model3DViewer.tsx [hunk 1]
```diff
@@ -217,7 +218,10 @@ export class Model3DViewer extends React.Component<Model3DViewerProps> {
 
   render() {
     return (
-      <div style={{ width: '100%', height: '100%' }} ref={this.divWrapper} />
+      <div
+        style={{ width: '100%', height: '100%', fontSize: 0 }}
```
Comment (f1cognite) [inline src/components/Model3DViewer/Model3DViewer.tsx]: fontSize?

File: src/components/Model3DViewer/Model3DViewer.tsx [hunk 2]
```diff
@@ -253,9 +257,7 @@ export class Model3DViewer extends React.Component<Model3DViewerProps> {
 
       const reusableBox = new THREE.Box3();
 
-      // @ts-ignore
-      const { matrixWorld } = this.model;
-      const bb = this.model.getBoundingBox(nodeId, reusableBox, matrixWorld);
+      const bb = this.model.getBoundingBox(nodeId, reusableBox);
```
Comment (f1cognite) [inline src/components/Model3DViewer/Model3DViewer.tsx]: Nice

File: src/components/Model3DViewer/stories/full.md [hunk 3]
```diff
@@ -4,10 +4,10 @@
 
 #### Requirements:
 
-To use this component you have to install `@cognite/3d-viewer` package (version 4.1.3):
+To use this component you have to install `@cognite/3d-viewer` package (version 5.0.7):
```
Comment (f1cognite) [inline src/components/Model3DViewer/stories/full.md]: Or  5.x.x?

File: src/components/Model3DViewer/stories/full.md [hunk 4]
```diff
@@ -4,10 +4,10 @@
...
 
 ```bash
-yarn add @cognite/3d-viewer@4.1.3
+yarn add @cognite/3d-viewer@5.0.7
```
Comment (f1cognite) [inline src/components/Model3DViewer/stories/full.md]: Not only yarn add @cognite/3d-viewer@5 ?

File: src/components/Model3DViewer/Model3DViewer.tsx (hunk 1 above)
Comment (mrtraser) [reply src/components/Model3DViewer/Model3DViewer.tsx]: yes, its related to @cognite/3d-viewer `_resizeIfNeeded` function, which is check wrapper dimension and fit canvas to it (if they are different). But after changing of canvas size, dimension of wrapper also grows, so it cause infinite increasing of canvas. If font-size of wrapper is 0, it has the same size as canvas and doesn't trigger `_resizeIfNeeded`

File: src/components/Model3DViewer/stories/full.md (hunk 3 above)
Comment (mrtraser) [reply src/components/Model3DViewer/stories/full.md]: yes, fixing

File: src/components/Model3DViewer/stories/full.md (hunk 4 above)
Comment (mrtraser) [reply src/components/Model3DViewer/stories/full.md]: true, doing

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-219.gearboxjs.preview.cogniteapp.com

**Pull request merged**